packages/schemas/src/types/jwt-customizer.ts

```diff
@@ -37,7 +38,7 @@ export const jwtCustomizerUserContextGuard = userInfoGuard.extend({
       ),
     })
   ),
-  organizations: z.record(organizationDetailGuard),
+  organizations: z.array(organizationDetailGuard),
```

Comment:
will it be better to align the format with id token? i.e. `organizations` and `organizationRoles`

Reply:
Updated.